docs/2026-06-25_aws_gateway_benchmark/scripts/summarize.py

```
@@ -0,0 +1,296 @@
+    print(f"(latency = median across {len(runs)} trial(s); p99/rps show [min–max])")
+
+    # ── Latency (median across trials) ─────────────────────────────────────────
+    base_p50 = {}
+    for dialect, mode in VARIANTS:
+        b = agg_variant(collect(runs, "baseline", dialect, mode))
+        base_p50[(dialect, mode)] = b.get("p50")
+
+    print("\nLATENCY  (ms; rps = completed req/s @ c={})".format(meta.get("concurrency", "?")))
+    hdr = (f"{'target':9} {'variant':18} {'ok/fail':>11} {'rps':>7} {'p50':>7} "
+           f"{'p99':>7} {'p99 range':>15} {'ttft':>7} {'ovhd':>7}")
+    print(hdr); print("-" * len(hdr))
```

Ruff (0.15.18)

[error] Multiple statements on one line (semicolon)

(E702)